.travis.yml

```diff
@@ -23,11 +23,11 @@
   before_install:
     - curl -sL https://github.com/jpm4j/jpm4j.installers/raw/master/dist/biz.aQute.jpm.run.jar >jpm4j.jar
     - java -jar jpm4j.jar -u init
-    - ~/jpm/bin/jpm install com.codacy:codacy-coverage-reporter:assembly
+  #  - ~/jpm/bin/jpm install com.codacy:codacy-coverage-reporter:assembly
```

Comment:
In general do not commit commented code. If this is broken it should be removed and we'll add it back later.